code/modules/projectiles/projectile/beams.dm

```diff
@@ -26,7 +26,7 @@
 	eyeblur = 0
 
 /obj/item/projectile/beam/pistol
-	damage = 25
+	damage = 20
```

Comment:
Undocumented change

Reply:
I think I messed this up, I was trying to make sure this pistol wasn't too op. I'll fix that.